### ISSUE TITLE
[Feature Request] Search/Filter folders by pattern alongside regex

### ISSUE THREAD

**Issue by tomas-chrastina** [alainfrisch/nostalgy#45]
Hi,

it would be nice to add option to choose how to search for destination folder. Currently, it uses **regex**. 

So I can "filter" results by useing **regex**:
`Move message to: Company.*Task`

But sometimes it would be easier (maybe matter of habbit), if I could use **pattern**, so I could achive the same this way:
`Move message to: Company*Task`

Thank you in advance for considering this.

--
Best regards,
Tomas Chrastina.

**Comment by caugner** [comment]
I agree that supporting both **regex** and **wildcard** patterns would be desirable.

In order to make Nostalgy approachable for users that don't know regex, I would even suggest to make **wildcard* the default (except for existing users, if possible) and add a `Match using Regular Expression` preference.

**Comment by tomas-chrastina** [comment]
I agree, and for now using `..` instead of `.*` works for me just fine.
But to have more options for different kind of users is a good thing.
